### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Calamity210** [Calamity210/BirbLang#1]
**Describe the bug**
Cant access `List`s or `Map`s from within a class directly

**To Reproduce**
1. Launch birb shell with the `birb` command
2. Type 
```dart
class Foo {
   List list = ["value"];
}

screm(Foo.list[0]);
```

3. Run program with `runBirb` command

**Expected behavior**
Expected to print `value`.

**Additional context**
The scope gets messed up and the program tries to find a list within the global scope.

**Comment by Calamity210** [comment]
Was an error of not passing the correct scope to list access types

**Comment by Calamity210** [comment]
corrected in 61d1a25
